ci/npm-publish.yml

```diff
@@ -28,7 +28,15 @@ jobs:
         with:
           node-version: 12
           registry-url: https://registry.npmjs.org/
-      - run: npm publish
+          scope: '@your-github-username'
```

Comment:
I think the scope setting is not required for `registry.npmjs.org`.

Reply:
That's true, just my preference. Reverted